students/20th/hwisungkim/js/login.js

```diff
@@ -0,0 +1,47 @@
+const thisIsId = document.getElementsByClassName("id_input")[0];
+const thisIsPw = document.getElementsByClassName("pw_input")[0];
+const login_btn = document.getElementById("login_btn");
+
+login_btn.addEventListener('click',function(e){// 아이디,비밀번호 상자 입력할때마다 검사
+    alert(access_permission()); //로그인함수
+})
+
+thisIsId.addEventListener('keyup',function(e){   //아이디입력칸 타자칠때마다 길이체크
+    check_id_pw(); //로그인버튼 온오프함수
+});
+
+thisIsPw.addEventListener('keyup',function(e){
+    check_id_pw();
+});
+
+function access_permission(){
+    let id=document.getElementById("id_input").value; //입력된 아이디값
+    let pw=document.getElementById("pw_input").value; //입력된 비번값
+    return (id===pw ? "로그인 되었습니다" : "비밀번호가 틀렸습니다");
+}
```

Comment:
```suggestion
function accessㅖermission () {
    return id.value === pw.value ? "로그인 되었습니다" : "비밀번호가 틀렸습니다";
}
```
- 띄어쓰기 적절히 사용해주시면 좋습니다.
- 이미 id, pw를 셀렉트 하는 로직이 최상위 스코프에 정의되어 있었기 때문에 해당 내용을 재사용하면 될 것 같네요 :)
- 삼항연산자 잘 사용해주셨습니다!
- let보다는 가급적 const를 사용하는 것이 좋습니다. 그 이유에 대해 알고 계시다면 댓글로 남겨주세요!

Reply:
아하 최상단에 제가 아이디와,비밀번호의 elements의 속성값을 가져와서 const로 만든 변수에 할당했었는데
거기에 .value 를 붙이면 그대로 재사용 가능했었네요! 
let은 재선언이 가능해서 코드가 많아지면  자신도 모르게 중복되게 사용할수있어 오류를 야기한다고 배웠습니다!